axelrod/strategies/finite_state_machines.py

```diff
@@ -571,3 +571,130 @@ def __init__(self) -> None:
         )
 
         super().__init__(transitions=transitions, initial_state=0, initial_action=C)
+
+
+## Strategies trained with Moran process objectives
+
+
+class TF1(FSMPlayer):
+    """
+    A FSM player trained to maximize Moran fixation probabilities.
+
+    Names:
+
+        - TF1: Original name by Marc Harper
```

Comment:
We should remember to include the moran paper as a reference for this once we have a title etc...